packages/web/src/components/gcds-fieldset/gcds-fieldset.stories.tsx

```diff
@@ -56,16 +56,16 @@ export default {
   },
 };
 
-const Template = (args) => `
+const Template = (args) => (`
 // Web Component (Angular, Vue)
 <gcds-fieldset
   fieldset-id="${args.fieldsetId}"
   legend="${args.legend}"
-  hint="${args.hint}"
-  error-message="${args.errorMessage}"
-  required="${args.required}"
-  disabled="${args.disabled}"
-  lang="${args.lang}"
+  ${args.hint ? `hint="${args.hint}"` : null}
```

Comment:
The hint is not required, but it shows up by default. Is that on purpose to encourage people to use a hint?